internal/logger/logger.go

```
@@ -0,0 +1,56 @@
+package logger
+
+import (
+	"os"
+
+	"github.com/hashicorp/go-hclog"
+)
+
+var log hclog.Logger
+
+func Init(verbose bool, debug bool, quiet bool) {
+	level := hclog.Info
+	if quiet {
+		level = hclog.Error
+	} else if debug {
+		level = hclog.Debug
+	} else if verbose {
+		level = hclog.Trace
+	}
+
+	log = hclog.New(&hclog.LoggerOptions{
+		Name:       "tf-migrate",
+		Level:      level,
+		Output:     os.Stderr,
+		JSONFormat: false,
+		Color:      hclog.AutoColor,
+	})
+}
+
+func Debug(msg string, args ...interface{}) {
+	if log == nil {
```

These package methods are bad because they fail silently. It's better to create the logger in `func main` and pass it around as any other dependency.